counterfit/targets/digits_keras.py

```diff
@@ -0,0 +1,128 @@
+import os
+
+import numpy as np
+import tensorflow as tf
+from counterfit.core.targets import CFTarget
+from tensorflow import keras as K
+
+# for ART
+tf.compat.v1.disable_eager_execution()
+
+class DigitKeras(CFTarget):
+    data_type = "image"
+    target_name = "digits_keras"
+    task = "classification"
+    endpoint = "digits_keras/mnist_model.h5"
+    input_shape = (28, 28, 1)
+    output_classes = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"]
+    classifier = "keras"
```

Comment:
`classifier` should be "blackbox".
![image](https://user-images.githubusercontent.com/43563047/199318433-d5c659db-cdc6-47a9-b402-03fe9ed8920e.png)

Reply:
@rdheekonda, great catch! Fixed in f1501b6de36a49666ae85c0d470601dd42120e1e